executor/builder.go

```diff
@@ -274,6 +278,22 @@ func (b *executorBuilder) buildLimit(v *plan.Limit) Executor {
 	return e
 }
 
+func (b *executorBuilder) buildUnion(v *plan.Union) Executor {
+	e := &UnionExec{
+		fields: v.Fields(),
+		Sels:   make([]Executor, len(v.Selects)),
+	}
+	for i, sel := range v.Selects {
+		selExec := b.build(sel)
```

Comment:
It will be checked by caller, and there is only one kind of error "ErrUnknownPlan"